### ISSUE TITLE
[question] How to load hific iamge directly？

### ISSUE THREAD

**Issue by cczutang** [Justin-Tan/high-fidelity-generative-compression#22]
Do we need a decoder to parse the hific image?

**Comment by Justin-Tan** [comment]
Yes, images are encoded in `.hific` format - to decode this back to an image you need to first:

1. Decode the bitstream to the latent representation using a standard entropy (here we use ANS).
2. Pass the latent representation through the NN decoder/generator - this is the bottleneck.

Using `compress.py` will do both for you automatically provided you pass in the right flags. Check the [usage guide](https://github.com/Justin-Tan/high-fidelity-generative-compression/blob/master/src/README.md) for more info about this.
